draft-irtf-cfrg-aead-limits.md

```diff
@@ -533,11 +543,24 @@ v * 2^64 + (2l * (v + q))^2 <= (p / u) * 2^128
 
 # Security Considerations {#sec-considerations}
 
-Many of the formulae in this document depend on simplifying assumptions that are
-not universally applicable.  When using this document to set limits, it is
-necessary to validate all these assumptions for the setting in which the limits
-might apply.  In most cases, the goal is to use assumptions that result in
-setting a more conservative limit, but this is not always the case.
+Many of the formulae in this document depend on simplifying assumptions,
+from differing models, that are not universally applicable. When using this
+document to set limits, it is necessary to validate all these assumptions
+for the setting in which the limits might apply. In most cases, the goal is
+to use assumptions that result in setting a more conservative limit, but this
+is not always the case. As an example of one such simplification, this document
+defines v as the total number of failed decryption queries, whereas in certain
+models, such as {{GCMProofs}}, it is defined as the total number of queries.
```

Comment:
This is actually an interesting point: Yes, {{GCMProofs}} defines v as total number of decryption queries. However, forwarding encrypted ciphertexts to the decryption oracle or repeating queries to the decryption oracle is not allowed:

"Without loss of generality, we assume that A does not make trivial queries:
if the encryption oracle returns (C, T ) for a query (N, A, M ), then A does not make a query
(N, A, C, T ) to the decryption oracle, and A does not repeat a query to the decryption oracle." (p. 5)

That means that, in practice, any decryption query is a forgery _attempt_, so the difference actually is that we count _failed_ forgery attempts (which is the only thing you can do in practice) while they count _all_ forgery attempts (which you can only do in a model).  Of course, in practice, we care about bounding the any successful forgery.

Does the following capture this?

```suggestion
is not always the case. As an example of one such simplification, this document
defines v as the total number of failed decryption queries (that is, failed forgery
attempts), whereas models usually count in v all forgery attempts.
```